sessions/js-functions-2/greeting-page/js/index.js

```diff
@@ -0,0 +1,33 @@
+console.clear();
+
+/*
+Change the contents of this page depending on the current day and time.
+
+- Write a function getGreeting that returns a different greeting depending on the current time:
+  - 6-12am: returns "Good Morning"
+  - 12am - 6pm: returns "Good Afternoon"
+  - 6pm - 10pm: returns "Good Evening"
+  - 10pm - 6am: returns "Good Night"
+
+- Write a function getDayColor that returns a different color depending on the current weekday:
+  - monday: "darkgray"
+  - tuesday - friday: "lightblue"
+  - saturday - sunday: "hotpink"
+
+(HINT: you can get the current time: new Date().getHours() )
```

Comment:
If the students only use the hours, then we have to change the exercises, for example:
```
  - 6am-12pm: returns "Good Morning"
  - 13pm - 6pm: returns "Good Afternoon"
  - 7pm - 10pm: returns "Good Evening"
  - 11pm - 5am: returns "Good Night"
```
It is unclear whether the students should only use hours, or whether they should use minutes and seconds, so for example:
11:59:59pm => "Good Morning"
12:00:00pm => "Good Morning" or "Good Afternoon"??
12:00:01pm => "Good Afternoon"
Also it is unclear whether for example the exact time 12:00:00pm should give the greeting "Good Morning" or "Good Afternoon"